content/00.zksync-network/50.sdk/10.js/01.ethers/04.guides/04.interop-messages.md

```diff
@@ -0,0 +1,121 @@
+---
+title: Interop Messages
+description: A guide for sending and verifying interop messages.
+---
+
+::callout{icon="i-heroicons-information-circle" color="blue"}
+The `InteropClient` is only available on v6.
+::
+
+Interop messaging enables sending and verifying messages across ZKsync chains via ZKsync Gateway.
+It is the first phase of universal interoperability for the Elastic Network.
+
+An interop message consists of arbitrary data and has two simple properties:
+
+1. Anyone can send a message.
+1. Anyone can verify that a given message was successfully sent on some chain.
+
+The message itself has no destination chain or address.
+It’s simply a payload created by a user or contract that gets broadcast.
+There is no expiration for when messages can be verified.
+
+Messages can only be sent and verified on ZKsync chains that use ZKsync Gateway.
```

Comment:
Thoughts on clarifying, or if we have point to a table of who is on Gateway. The guide uses abstract testnet which works, but are not on gateway for mainnet which some devs might be confused about.